crates/containerd-shim-wasm/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ## [Unreleased]
 
+## [v0.9.0] - 2025-01-27
```

Comment:
I don't see note of #775 in the change log below. Even though it is internal change I think we should mention it in the release notes?

Reply:
Good call I will add it

Reply:
Sorry, I'm still struggling with updating the changelog 🤦‍♂️